scripts/src/admin/services.ts

```diff
@@ -689,3 +692,15 @@ export async function changeOffer(body: ChangeOfferData, params: { offer_id: str
 
 	await offer.save();
 }
+
+export const updateAppConfig = async function(req: Request, res: Response) {
+	const app = (await Application.findOne({ id: req.params.application_id }))!;
+	try {
+		app.config = req.body;
```

Comment:
maybe define a type for this specific request so it will be clear?

Reply:
Can you pls guide me what Request type fits here? Or maybe I need to create one?

Reply:
you can do:
```
export type UpdateAppConfigRequest = Request & {
	body: {
		...
	}
}

export const updateAppConfig = async function(req: UpdateAppConfigRequest, res: Response) {
	...
```